perfkitbenchmarker/pkb.py

```diff
@@ -114,6 +113,12 @@
                     'characters in length.')
 flags.DEFINE_string('owner', getpass.getuser(), 'Owner name. '
                     'Used to tag created resources and performance records.')
+flags.DEFINE_boolean('enable_detailed_timing', False, 'Includes detailed '
```

Comment:
Converted it to a list in 30b489d. Runtimes and timestamps can be specified individually.